edgelet/dps/src/registration.rs

```diff
@@ -3,7 +3,6 @@
 use std::sync::{Arc, RwLock};
 use std::time::{Duration, Instant};
 
-use base64;
```

Comment:
what happened? why has the build started to fail all of a sudden? Some problems are related to new rust update. But some - not

Reply:
I needed to fix clippy errors as this PR https://github.com/Azure/iotedge/pull/2871 failed check-in build.  I think we use latest clippy (nightly?) which causes checkin build failed.

Reply:
we use stable clippy. in the latest update, the new wildcard check was added. but it doesn't explain why unused imports passed previously